contracts/test/Gateway.t.sol

```diff
@@ -858,13 +861,25 @@ contract GatewayTest is Test {
                 SetTokenTransferFeesParams({
                     assetHubCreateAssetFee: createTokenFee * 2,
                     registerTokenFee: registerTokenFee,
-                    assetHubReserveTransferFee: sendTokenFee
+                    assetHubReserveTransferFee: sendTokenFee * 3,
+                    destinationMaxTransferFee: maxDestinationFee + 1
```

Comment:
Why + 1 here?

Reply:
Arbitrary value just used to test. Just want to see the value changed.